tests/erlang_tests/test_lists_keyfind.erl

```diff
@@ -0,0 +1,71 @@
+-module(test_lists_keyfind).
```

Comment:
We need to add copyright headers, as in all other tests, so we can pass REUSE checks. Without them REUSE CI check will fail. This comment applies to all other test files that have been introduced here.